src/http/http_sig.h

```diff
@@ -112,6 +112,7 @@ namespace http
 
     const auto signature = kp->sign(to_sign.value());
 
+    // TODO: Should insert correct keyId, but we don't have a cert here!
```

Comment:
Our C++ signer produces signatures we probably shouldn't be accepting. But because they _also_ come with a TLS handshake identity, everything works. This is real bad for audit - the endpoint thinks it has (and stores) a signed request that doesn't let anyone verify the request. As this TODO notes, this is non-trivial to fix because we only have a key-pair at this point, not the specific _cert_ that we treat as the caller's identity. Perhaps an argument for using public keys as identities rather than certs? But we can plumb through the needed value for this instance.

Reply:
Tracking this in #2018, so I'll remove the TODO here.